<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dodaj_alarm</class>
 <widget class="QDialog" name="Dodaj_alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj System Alarmowy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="n_nazwa_modelu">
       <property name="text">
        <string>Nowy model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Klasa systemu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="n_klasa_systemu">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>   0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Pobór mocy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="n_pobor_mocy">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>   0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Liczba zgłoszeń</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="n_liczba_zgloszen">
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>   0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Przycisk napadowy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="n_przycisk_napadowy">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nazwa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="n_nazwa_systemu">
       <property name="text">
        <string>Nowy System Alarmowy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Stan: WYŁ/WŁ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="n_stan">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Zasilacz awaryjny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="n_zasilacz_awaryjny">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>n_pobor_mocy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>n_liczba_zgloszen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>n_klasa_systemu</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
